cmd/boulder-va/gsb_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/test"
+)
+
+func TestConfigCheck(t *testing.T) {
```

Comment:
We should have test cases to cover the `new...` and `IsListed` functions as well.